### PR TITLE
add new crypto sites

### PR DESCRIPTION
**Pull request by allensakari**
add popular crypto sites into catagory.

### CODE REVIEW

File: data/binance [hunk 1]
```diff
@@ -28,3 +28,7 @@ binancezh.top
 # API
 binanceapi.com
 bnbstatic.com
+
+# saas 
+saasexch.com
+appsflayer.com
```
Comment (rootmelo92118) [inline data/binance:34]: Could you provide any evidence to prove this domain is belonging to `binance`?

File: data/category-cryptocurrency [hunk 2]
```diff
@@ -16,23 +16,31 @@ include:zb
...
+coinmarketcap.com
+crypto.com
 cryptocompare.com
 curve.fi
 debank.com
 dogecoin.com
+gate.io
 infura.io
+mexc.com
+mexcsensors.com
```
Comment (rootmelo92118) [inline data/category-cryptocurrency:38]: Could you provide any evidence to prove this domain is belonging to this category?

**Review by rootmelo92118 — APPROVED** [review]

File: data/binance (hunk 1 above)
Comment (rootmelo92118) [inline data/binance:34]: Could you provide any evidence to prove this domain is belonging to `binance`?

File: data/category-cryptocurrency (hunk 2 above)
Comment (rootmelo92118) [inline data/category-cryptocurrency:38]: Could you provide any evidence to prove this domain is belonging to this category?

**Review by rootmelo92118 — CHANGES_REQUESTED** [review]

**Comment by rootmelo92118** [comment]
@allensakari

**Comment by allensakari** [comment]
> @allensakari

sorry, here are some reference screenshots.
mexc
saasex com

**Review by rootmelo92118 — APPROVED** [review]

**Pull request merged**